packages/react/popover/src/Popover.tsx

```diff
@@ -246,69 +314,31 @@ const PopoverContentImpl = React.forwardRef((props, forwardedRef) => {
   // the last element in the DOM (beacuse of the `Portal`)
   useFocusGuards();
 
-  // Hide everything from ARIA except the content
-  React.useEffect(() => {
-    const content = contentRef.current;
-    if (content) return hideOthers(content);
-  }, []);
-
   return (
     <PortalWrapper>
```

Comment:
This differs from `Dialog` in that we offer the ability to not portal, that said it probably will need to be duplicated in both modal/non-modal versions because of this: https://github.com/radix-ui/primitives/pull/699#discussion_r648391771